srcpkgs/sirikali/template

```diff
@@ -0,0 +1,24 @@
+# Template file for 'sirikali'
+pkgname=sirikali
+version=1.5.0
+revision=1
+#archs="i686 x86_64"
+wrksrc="SiriKali-${version}"
+create_wrksrc=yes
+build_style=cmake
+configure_args="-DCMAKE_INSTALL_PREFIX=/usr -DQT5=true
+ -DCMAKE_BUILD_TYPE=RELEASE"
+#make_build_args=""
+#make_install_args=""
+#conf_files=""
+#make_dirs="/var/log/dir 0755 root root"
+hostmakedepends="qt5-qmake qt5-host-tools pkg-config qt5-devel"
+makedepends="libgcrypt-devel libpwquality-devel
+ libsecret-devel $(vopt_if kde kwallet-devel)"
```

Comment:
this build option is not defined anywhere in the template

Reply:
also, line wrapping is only needed at ~80 characters, this could probably fit on one line